content/terraform/v1.12.x/data/language-nav-data.json

```
@@ -109,7 +107,7 @@
         ]
       },
       {
-        "title": "The <code>terraform_data</code> Resource Type",
+        "title": "Create a <code>terraform_data</code> resource",
```

```suggestion
        "title": "<code>terraform_data</code> resource reference",
```
This is really the reference for this specific type. It's kind of awkward as a one-off right here, but I don't know that there is a better solution until we do some even more comprehensive reorganization.  @BrianMMcClain @robin-norwood, what if we move this page to the reference section for now?